exporter/prometheus/prometheus.go

```diff
@@ -255,14 +255,6 @@ func tagKeysToLabels(keys []tag.Key) (labels []string) {
 	return labels
 }
 
-func tagsToLabels(tags []tag.Tag) []string {
-	var names []string
-	for _, tag := range tags {
-		names = append(names, internal.Sanitize(tag.Key.Name()))
```

Comment:
Why did you remove this? It is imperative that we sanitize keys for Proemtheus otherwise "org/foo" will fail for Prometheus which expects "org_foo".

Reply:
This is unused as you can see I did not change any code that calls it, and this seems to be a package private function.

Reply:
Gotcha, but it is noisy in this PR and unrelated to this change.

Reply:
I see your point, but dead code is the thing that I "like" the most :) happy to do a different PR if you feel strong.

Reply:
Reverted and I would send a different PR.